homeassistant/components/husqvarna_automower_ble/strings.json

```diff
@@ -4,18 +4,42 @@
     "step": {
       "user": {
         "data": {
-          "address": "Device BLE address"
+          "address": "Device BLE address",
+          "pin": "Mower PIN"
+        },
+        "data_description": {
+          "pin": "The PIN used to secure the mower"
```

Comment:
My mower does not have a pin, what should I enter here? Maybe we describe it here.

Reply:
Note: I had the same comment, but it's hidden away above https://github.com/home-assistant/core/pull/135440#discussion_r2256781239

Reply:
OK. Unless we can reliably detect if the mower needs PIN (and I think your answer suggests we can't), we should go ahead with the PR as is.